browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -258,6 +262,20 @@ void BraveAppMenuModel::BuildBrowserSection() {
   }
 }
 
+void BraveAppMenuModel::BuildAutofillSubMenu() {
+  ui::SimpleMenuModel* autofill_menu_model =
+      static_cast<ui::SimpleMenuModel*>(GetSubmenuModelAt(
+          GetIndexOfCommandId(IDC_PASSWORDS_AND_AUTOFILL_MENU).value()));
+  DCHECK(autofill_menu_model);
```

Comment:
is it an issue if this var doesn't get set? Thinking CHECK or error handling with a nil return might be better here if so instead of DCHECK which I believe gets optimized out at build time unless it's a debug build

Reply:
I think in general we're meant to prefer using `CHECK` to `DCHECK` now